test/apiTest.js

```diff
@@ -1,58 +1,79 @@
 // next we will write an API test for '/breed routes' GET and GET by id: 
 
 const request = require('supertest');
-const app = require('../src/app');
+const should = require('should');
+
+const buildServer = require('../src/app'); 
+const generateTestId = {
+    generate () {
+        return "tmk60ux2b" 
+
+    } 
+}
+    
+const appTest = buildServer(generateTestId);
 
 //==================== user API tests ====================
 
 
 describe('GET /cats', function () {
-    it('responds with json object containing a list of cat ids and name only', function (done) {
-        request(app)
+    it('responds with json object containing a list of cats with id & name properties only', function (done) {
+        request(appTest)
             .get('/cats')
             .set('Accept', 'application/json')
             .expect('Content-Type', /json/)
+            .expect(function (res) {
```

Comment:
it'd be good to clean up this area - there seem to be a lot of comments which might not be fully resolved yet?